config/routes.rb

```diff
@@ -7,7 +7,7 @@
 
   resources :posts, only: [:new, :create]
   resource :user_session, only: [:new, :create, :destroy]
-  resources :users, only: [:new, :create] do
+  resources :users, only: [:new, :create, :show, :edit, :update] do
```

Comment:
This can be flipped from `only: [:new, :create, :show, :edit, :update]` to `except: [:index, :destroy]`